documentation/src/docs/asciidoc/docinfos/index-docinfo.html

```diff
@@ -0,0 +1 @@
+<link rel="icon" type="image/png" href="{{ site.baseurl }}/img/junit5-logo.png">
```

Comment:
No, it does not work locally. I'm really certain that you're browser is fooling you. Please try a different one.

Regarding the Javadoc: We could [filter them](https://docs.gradle.org/current/userguide/working_with_files.html#sec:filtering_files) while copying them to the docs folder and replace `<head>` with `<head><link rel="..."...>`.